### PR TITLE
448 redux

### PR DESCRIPTION
**Pull request by DevonianTeuchter**
Re-PRing Promote changes.  Branch & PR 448-zopen-clone-spits-out-errors ended up a merge-mess after files moved!
This PR takes the changes from the previous branch and reapplies them over main.

### CODE REVIEW

File: bin/zopen-init [hunk 1]
```diff
@@ -248,284 +234,155 @@ fi
...
-printVerbose "Ensuring correct uid [$effuid] & gid [$effgid] for configfile if !installing as user (ie su/root/etc)"
-chown "$effuid" "$configFile" 2>/dev/null # suppress if not permitted
-chgrp "$effgid" "$configFile" 2>/dev/null # suppress if not permitted
+writeConfigFile "$configFile" "$rootfs" "$zopen_pkginstall" "$ZOPEN_CA"
 printInfo "- Created config in $configFile."
 
+export ZOPEN_ROOTFS="$rootfs"
 printInfo "- Root FS available at $rootfs"
+# Generate any system values that can be user changed
+echo "15" > "$rootfs/etc/zopen/rm_fileprocs"
```
Comment (IgorTodorovskiIBM) [inline bin/zopen-init:248]: What does 15 represent here? Number of processes?

Comment (DevonianTeuchter) [reply bin/zopen-init:248]: It's used in the unmerge processing where the FS is scanned for obsolete links. 15 files per "thread" (or rather process); seemed to give a reasonable amount of workload distribution without spawning too many simultaneous processes for things like nurses with it's 4000+ files). That generally does need a speed up though...

**Review by MikeFultonDev — APPROVED** [review]
LGTM!

**Review by IgorTodorovskiIBM — APPROVED** [review]
LGTM

**Comment by MikeFultonDev** [comment]
@DevonianTeuchter now that we have a test framework can you write tests for the new function?

**Comment by DevonianTeuchter** [comment]
Already working on them :-)

**Pull request merged**